root/static/scripts/relationship-editor/common/dialog.js

```diff
@@ -326,6 +335,11 @@ const PART_OF_SERIES_LINK_TYPE_GIDS = _.values(PART_OF_SERIES_LINK_TYPES);
             this.showLinkTypeHelp(!this.showLinkTypeHelp.peek());
         },
 
+        linkTypeName: function () {
```

Comment:
In your screenshot, the link type name shown in the string ("composed") is different from the one in the `<select>` ("composer"), which reads oddly. I guess it's something to do with the `backwards` argument to `linkPhrase.clean`. (Backwards = the target is entity0.) I'd see how the select options handles it.